src/amo/components/LandingPage.scss

```diff
@@ -3,3 +3,69 @@
 .LandingPage {
   padding: $padding-page;
 }
+
+.Addon-Header {
+  margin: 10px;
+  max-width: 100%;
+  text-align: center;
+}
+
+.Icon-persona {
```

Comment:
These icon styles can go into the `Icon` component's styles. The icon files themselves can be moved next to the component as well.